src/App.css

```diff
@@ -1,38 +1,32 @@
-.App {
-  text-align: center;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+html, body {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  line-height: 1.7em;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+a {
+  text-decoration: none;
+  color: inherit;
```

Comment:
Please add this CSS into a separate CSS file. That way we can more clearly tell which styles are coupled to which components/pages. It should look like this

> components
>> Component 1
>>> Component1.js
>>> Component1.css

> pages
>> Page1
>>> Page1.js
>>> Page1.css

Reply:
I add these css inside of NavBar.css and bring the red part back.
And I put NavBar.js and NavBar.css inside of a folder under components for coupling.
I will push code sooner.